### DISCUSSION TITLE
Exporting on a Monthly Basis

### DISCUSSION THREAD

**Discussion by devnull044** [ReagentX/imessage-exporter#338] [Q&A] [answered]
Im trying to get an understanding on how to continuously use the tool on a monthly bases. For example, I want to pull messages at the end of every month (its for one contact but that feature isn't enabled yet ☹️.) 

How would I go about doing this? As of right now, I have to move the existing export to a separate folder because it won't reexport if there is already one present. I hope my question is clear and makes sense.

**Comment by ReagentX** [comment, marked as answer]
You can use a tool like [`cron`](https://man7.org/linux/man-pages/man5/crontab.5.html) to schedule exports and use the `-o` flag to specify a different folder for each export.

**Reply by devnull044** [reply]
So this would essentials create a new database every time correct?

**Reply by ReagentX** [reply]
`imessage-exporter` doesn't create any databases, it connects to an existing database and exports the data to the specified directory. You can use the `-s` and `-e` to restrict messages to the month you want to export. More information about how to use the app is [here](https://github.com/ReagentX/imessage-exporter/tree/develop/imessage-exporter#how-to-use).
